course_catalog/utils.py

```diff
@@ -270,3 +270,64 @@ def safe_load_json(json_string, json_file_key):
     except rapidjson.JSONDecodeError:
         log.exception("%s has a corrupted JSON", json_file_key)
         return {}
+
+
+def parse_instructors(staff):
+    """
+    Parses staff/instructors users including their full name, salutation etc
+
+    Args:
+        array (dict): staff/instructors
+
+    Returns:
+        array (dict): parsed instructors
+    """
+    instructors = []
+    for person in staff:
+        instructor = {
+            "first_name": person.get("given_name", person.get("first_name")),
+            "last_name": person.get("family_name", person.get("last_name")),
+            "full_name": person.get("title"),
+        }
+
+        if person.get("salutation"):
+            if instructor.get("full_name") and (
+                not instructor.get("full_name").startswith(person.get("salutation"))
+            ):
+                instructor["full_name"] = "{salutation} {full_name}".format(
+                    salutation=person.get("salutation"),
```

Comment:
I noticed while importing one course - `res-ll-003-build-a-small-radar-system-capable-of-sensing-range-doppler-and-synthetic-aperture-radar-imaging-january-iap-2011` - the salutation had a trailing whitespace character ("Prof. ") so the full name ended up having two whitespace characters between the salutation and first name ("Prof. &nbsp; &nbsp;Jeffrey S. Herd") - so might be good to use `.strip()` here:

```python
 salutation=person.get("salutation", "").strip(),
```

Reply:
Good catch! I also thought about this as salutations usually have a trailing whitespace, but I didn't find any after seeing some courses.
Anyhow, I'll update this PR.